### PR TITLE
bugfix-exlm-287

### PR DESCRIPTION
**Pull request by adobenitesh**
Jira ID: EXLM-287

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management
- After: https://bugfix-exlm-287-header--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -160,6 +193,11 @@
     height: 64px;
   }
 }
+@media (width <= 1024px) {
```
Comment (thomsebastin) [inline blocks/header/header.css:196]: @adobenitesh We decided initially to not use custom resolutions. Let's discuss to see if we can handle this within the agreed-upon values.

Comment (adobenitesh) [reply blocks/header/header.css:270]: I have added my inputs to EXLM-4 for custom breakpoint specific to header scenario.

File: blocks/header/header.js [hunk 2]
```diff
@@ -29,34 +32,61 @@ const communityTabContent = await fetchContent(
...
-/**
- * decorates the header, mainly the nav
- * @param {Element} block The header block element
- */
-export default async function decorate(block) {
-  const navWrapper = document.createElement('nav');
-  navWrapper.className = 'exl-topnav';
-  navWrapper.setAttribute('aria-label', 'Main navigation');
-  navWrapper.setAttribute('role', 'navigation');
+// REUSABLE VARIABLES
```
Comment (thomsebastin) [inline blocks/header/header.js:35]: I think this comment is not needed.

File: blocks/header/header.scss [hunk 3]
```diff
@@ -11,16 +11,69 @@
...
+    gap: 16px;
+  }
+
+  @media (width <= 1024px) {
+    background: var(--non-spectrum-bg);
+    display: block;
+    height: 100%;
+    left: -350px;
+    max-width: 350px;
+    overflow-y: scroll;
```
Comment (thomsebastin) [inline blocks/header/header.scss:34]: Won't this alway show a scrollbar on windows?

File: blocks/header/header.scss [hunk 4]
```diff
@@ -49,49 +103,103 @@
...
+    background-size: contain;
     display: inline-block;
-    height: 18px;
+    height: 64px;
+    margin: 0;
     vertical-align: middle;
     width: 18px;
     text-indent: -99999px;
+
+    @media (width <= 1024px) {
```
Comment (thomsebastin) [inline blocks/header/header.scss:125]: we have a mixin for standard breakpoints? Can we not leverage them here?

File: blocks/header/header.scss (hunk 3 above)
Comment (adobenitesh) [reply blocks/header/header.scss:34]: No, the scroll will be visible only when left nav slides to the right(clicking hamburger) and using nav items expand/collapse functionality

File: blocks/header/header.scss [hunk 5]
```diff
@@ -11,16 +11,69 @@
...
+    opacity: 1;
+    display: block;
   }
 }
 
 .header {
   border-bottom: 1px solid var(--spectrum-gray-200);
   font-family: var(--body-font-family);
   font-size: var(--spectrum-font-size-100);
+  height: 64px;
```
[the remark this answers is not in the document]
Comment (adobenitesh) [reply blocks/header/header.scss:61]: Done

File: blocks/header/header.scss [hunk 6]
```diff
@@ -11,16 +11,69 @@
...
+    background: var(--background-color);
+    position: fixed;
+    width: 100%;
+    left: 0;
+    top: 0;
+  }
+
+  &-wrapper {
+    position: relative;
+    z-index: 11;
```
[the remark this answers is not in the document]
Comment (adobenitesh) [reply blocks/header/header.scss:74]: We can review the approach.

File: blocks/header/header.scss (hunk 4 above)
Comment (adobenitesh) [reply blocks/header/header.scss:125]: Done

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**